packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -353,7 +353,9 @@ export class GPUPicker {
             return null;
         }
 
-        const { x: adjustedX, y: adjustedY, rttSizeW, rttSizeH } = this._prepareForPicking(x, y);
+        const { rttSizeW, rttSizeH, devicePixelRatio } = this._getRenderInfo();

```

Comment:
Is this file change expected? It should not be in that PR for sure ;)

Reply:
TBH I don't have a clue what is messed up here and why did this get into this PR. This file was changed by this PR: https://github.com/BabylonJS/Babylon.js/pull/16568

Reply:
I understand but it messes up with the history and if we have an issue or need to revert it will be a problem. Can you rebase with master maybe?

Reply:
> I understand but it messes up with the history and if we have an issue or need to revert it will be a problem. Can you rebase with master maybe?

Should be ok now.